SRC/NBASIC/nbas111e.txt

```diff
@@ -202,55 +202,54 @@ the functions' description.
 
 3.2. USING VRAM AS LOGICAL SEGMENTS
 
-NestorBASIC allows you to use the VRAM for emulate extra RAM segments. If 
-NestorBASIC founds a number S of segments, segment numbers 0 to S-1 refers to 
-RAM segments, as explained above. But segments S to S+7 or S+3 (according to 
-the VRAM capacity, 128K or 64K) are also available, and they refer to VRAM.
+NestorBASIC allows you to use VRAM to emulate extra RAM segments. If NestorBASIC
+found a number S of segments, segments 0 to S-1 correspond to RAM segments,
+as explained above. But segments S to S+7 or S+3 (depending on VRAM capacity,
+128K or 64K) are also available, and they refer to VRAM.
 
-Let's return to the previos sample. When installing NestorBASIC, the number 
-of available segments is 72. Then, segments 72 to 79 (if the computer has 
-128K VRAM) or 72 to 75 (if the computer has 64K VRAM) are available for VRAM 
-access.
+Let's go back to previos example. When NestorBASIC is installed resident, the   
+returned number of segments value is 72. Thus, segments 72 to 79 (if computer
+has 128K VRAM) or 72 to 75 (if computer has 64K VRAM) are VRAM segments.
 
-The correspondence between segments and VRAM address is reversed: the 
-segments with the lowest number refers to the highest VRAM addresses. This is 
-done in this way in order to help the user when discarding the VRAM segments 
-corresponding to the screen visualization: these segments will be the last 
+The correspondence between segments and VRAM addresses is reverse: the segments 

```

Comment:
I think "reversed" was correct here, as in "resversed relative to how RAM segments work". But it could be "the reverse" too (with an added "the").

Reply:
Ok.